src/test/resources/test-scripts/primitives_large_numbers.bsh

```diff
@@ -108,7 +108,7 @@ assertThat(t, instanceOf(Double.class));
 assertEquals(8.988465674311579E307, t, 0.0);
 t = 1.7976931348623167E616 / 2;
 assertThat(t, instanceOf(BigDecimal.class));
-assertEquals(8.9884656743115835E+615w, t);
+assertEquals(8.988465674311584E+615w, t);
```

Comment:
Huh? What's the hang up here?

Anyway... does intelij savvy `.bsh`?

Reply:
This is related to another change set where I set rounding mode for division. I am not entirely sure about that but it seems to avoid exception in case of inexact result or such.